scripts/dendrite_sytest.sh

```diff
@@ -62,6 +62,9 @@ rsync -r --ignore-missing-args --min-size=1B -av /work/server-0 /work/server-1 /
 if [ $TEST_STATUS -ne 0 ]; then
     # Build the annotation
     perl /sytest/scripts/format_tap.pl /logs/results.tap "$BUILDKITE_LABEL" >/logs/annotate.md
+    # If show-expected-fail-tests logged something, put it in the annotation
+    # We want nice output though, so surround the text output in code tags
```

Comment:
> surround the text output in code tags

where does this happen?

Reply:
Wow, this totally fell off my radar.

It's supposed to happen inside of `format_tap.pl`, which was *probably* supposed to be part of this PR, but perhaps I didn't push that change?

Time to go inspect my desktop...

Reply:
Aha, no they're here! https://github.com/matrix-org/dendrite/pull/870/files

Reply:
ok, but still, that means this comment is in the wrong place?

Reply:
Ah, yes quite.